plugins/modules/cluster_ha.py

```diff
@@ -452,7 +453,7 @@ def configure_ha(self):
 
                 try:
                     task = self.cluster.ReconfigureComputeResource_Task(cluster_config_spec, True)
-                    changed, result = wait_for_task(task)
+                    changed, result = wait_for_task(task, timeout=1)
```

Comment:
Why you've added timeout=1? Is there any reason for this? If yes, maybe worth to put comment.